### DISCUSSION TITLE
Demo Video for Sunbird RC usecases

### DISCUSSION THREAD

**Discussion by jagadishbabu** [Sunbird-RC/community#34] [Show & Tell]
@parthlawate @dileepbapat I am interested to circulate a demo video of the functioning of the Sunbird-RC project using demo workflows of a common use case to demonstrate the functioning of the system. Many people are asking for it. Can you please consider making a mock demo flow leveraging the current sunbird RC capabilities highlight each of the six verbs

**Comment by jagadishbabu** [comment]
I see a task in the project sheet - 

https://github.com/Sunbird-RC/community/projects/4#card-68373621)  - 

what is the ETA of this task?

**Reply by parthlawate** [reply]
@Renuka-S is working on the same. Renuka pls post an ETA.. Also in the guide deck that you use in the video, ensure its Verb driven.. Verb reference given by @jagadishbabu here https://github.com/Sunbird-RC/community/discussions/36

**Comment by Renuka-S** [comment]
Hi,

As mentioned on slack Paras is working on issue fixes once those are done
will complete the video. There have been schema and backend changes that
have affected these pages.
Issue List:

   1. Teacher - Send verification Experience
   2. Teacher - Academic qualifications details aren't visible
   3. Student - Send verification details for the sent data Especially
   institute name
   4. Institute - Attestation Taking first user for all records
   5. Student - Medium dropdown


@Parth Lawate ***@***.***> Could you provide access for
updating this ToDo list? Or you can move the Video ToDo into the "In
progress" bucket

On Tue, Sep 14, 2021 at 9:14 AM Parth Lawate ***@***.***>
wrote:

> @Renuka-S <https://github.com/Renuka-S> is working on the same. Renuka
> pls post an ETA.. Also in the guide deck that you use in the video, ensure
> its Verb driven
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Sunbird-RC/community/discussions/34#discussioncomment-1321641>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHV2GXN3SYLUGK2UADCCNB3UB3AKPANCNFSM5D7F3L2Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Best Regards,
Renuka N. Shikarkhane
Tekdi Technologies Pvt. Ltd.

**Reply by parthlawate** [reply]
you should be able to move to move the issues across buckets. are you getting any errors @Renuka-S ?

**Reply by jagadishbabu** [reply]
@parthlawate is the demo video completed?
